desc/protoprint/print.go

```diff
@@ -716,6 +716,10 @@ func (p *Printer) printMessageBody(md *desc.MessageDescriptor, mf *dynamic.Messa
 	}
 
 	skip := map[interface{}]bool{}
+	maxTag := int32(internal.MaxNormalTag)
```

Comment:
Super nit: You might want to turn this into a shared helper function `GetMaxTag(isMessageSet bool)` instead of having the public consts

Reply:
Yeah, that's a good idea.

Reply:
done